### PR TITLE
fixed more bugs

### PR DESCRIPTION
**Pull request by rachelsison**

### CODE REVIEW

File: client/js/components/create_form.jsx [hunk 1]
```diff
@@ -171,7 +173,7 @@ var CreateForm = React.createClass ({
                             </select>
             </div>
             <div id="SupportingFormFields">
-             {AppStore.currentQuestionType.map(function(questionType) {
+             {AppStore.currentQuestionType.map(function(questionType){
```
Comment (dduugg) [inline client/js/components/create_form.jsx]: :confused:

**Comment by dduugg** [comment]
:+1:

File: client/js/components/create_form.jsx (hunk 1 above)
Comment (owen-d) [inline client/js/components/create_form.jsx]: that face killed me, haha

**Comment by zindlerb** [comment]
:+1:

**Pull request merged**